docs/install.md

```diff
@@ -122,6 +122,45 @@ Select the following tabs for the installation description of your target.
 
     is needed. A recommended size is at least a free space of 10 GiB.
 
+    ??? important "Installation of VirtualBox with VirtualBox extension pack ("Guest additions")"
+        If you plan to install the VirtualBox extension pack, some things have to be taken into account to achieve a working time synchronization. In general:
+
+        - VirtualBox without extension pack needs activated time synchronisation from DietPi (systemd service)
+        - VirtualBox with extension pack needs deactivated time synchronisation from DietPi (systemd service) and some additional installation steps
+
+        The needed additional installation steps (use of the extension pack) are described here (installation steps on the host system) and below (installation steps within the DietPi guest system).  
+        The installation on the host system contains the base VirtualBox installation, the installation of the kernel headers and the installation of the extension pack. The VirtualBox host installation is described in the case of a Linux based host system.
+
+        1. Installation of VirtualBox on a Linux host system  
+           Following is the description of a manual installation (you need to change this depending on the actual VirtualBox release):
+
+             ```sh
+             cd ~
+             mkdir Downloads
+             cd Downloads
+             wget https://download.virtualbox.org/virtualbox/6.1.18/virtualbox-6.1_6.1.18-142142~Debian~buster_amd64.deb
+             apt --install-recommends install ./virtualbox-6.1_6.1.18-142142~Debian~buster_amd64.deb
+             ```
+
+            The installation option `--install-recommends` installs the kernel headers, otherwise you may run into time synchronization problems every time the kernel is updated.  
```

Comment:
So the kernel headers are required on the guest and the host? On the guest it's clear, as the guest additions are based on a kernel module driver, but I didn't know that the host requires an additional kernel module as well. I wonder how this is solved on Windows where no such is required.

I checked the `virtualbox` packages recommends:
```
Recommends: libasound2, libpulse0, libsdl-ttf2.0-0, kmod | kldutils | module-init-tools, linux-headers-generic | linux-headers-generic-pae | linux-headers-686-pae | linux-headers-amd64 | linux-headers-2.6-686 | linux-headers-2.6-amd64 | linux-headers, linux-image, gcc, make | build-essential | dpkg-dev, binutils, pdf-viewer
```
So if the kernel headers alone are sufficient, let's advice to install them explicitly to skip the bunch of additional dev libraries and build tools and have the instructions simpler as well:
```
apt install ./virtualbox-6.1_6.1.18-142142~Debian~buster_amd64.deb linux-headers-amd64
```
and below the note:
```
The kernel headers are required, otherwise you may run into time synchronization problems every time the host system kernel is updated.
```

Reply:
@MichaIng:
I am not 100% aware that the host needs the header files. I also wondered whether we can omit this. This especially also, because the command ./VBoxLinuxAdditions.run did report compilation problems.
Actually we could keep it as a relict to be sure on the host. But for now I will remove this, if we run into user problems, we can add it again.

Regarding the kernel header installation (see also in the guest system section):
I also thought about investigating which kernel header packages we have to install, because otherwise you get "a couple of MB" unneeded stuff installed. I did not investigate this further, so at least to be sure I propose to use  
`apt install linux-headers-amd64 build-essential`  
if we add this later again to the host installation.   
Q: What about `linux-headers-generic linux-headers`?  
Q: I assume `linux-headers-generic-pae inux-headers-686-pae` are not relevant because they rely to old 32bit PC systems we do not support, right?

Reply:
Agreed. Good to keep instructions simple, to not discourage users from installing guest additions at all when facing the too long list of tasks 😄.

`linux-headers-generic` is not a real package, probably it's an alias for the next matching headers package. Since we ship no i386 images, `linux-headers-amd64` is correct, matching the `linux-image-amd64` kernel package that we pre-install on all x86_64 images. Not exactly sure what PAE is about, but it indeed seems to be a 32-bit kernel/CPU feature. This exists for the 32-bit ARM kernel as well (https://packages.debian.org/buster/linux-image-armmp-lpae), but not for the 64-bit ARM kernel.